endaq/calc/stats.py

```diff
@@ -6,6 +6,356 @@
 
 import numpy as np
 import pandas as pd
+from scipy import signal
+import plotly.graph_objects as go
+
+from endaq.plot import rolling_min_max_envelope
+from endaq.calc import filters, integrate, utils, shock
+
+
+def shock_vibe_metrics(
+        df: pd.DataFrame,
+        tukey_percent: float = 0.1,
+        highpass_cutoff: float = None,
+        unit_conversion: typing.Literal["g_to_in", "g_to_mm", None] = None,
```

Comment:
Why not just make this numeric, and have 'constants' `G_TO_IN` and `G_TO_MM`? The user could do custom conversion by entering their own value.

Reply:
units will be a later thing to deal with, so I'm removing for now